mParticleCore.brs

```diff
@@ -1831,6 +1835,17 @@ function mParticleStart(options as object, messagePort as object)
                         eventAttributes.segment_duration = mediaSession.segment.duration.ToStr()
                     end if
                 end if
+
+                if (mediaSession.mediaSessionAttributes.count() > 0) then
+                    mputils = mparticle()._internal.utils
+                    mediaSessionAttributes = mediaSession.mediaSessionAttributes
+                    attributeKeys = mediaSessionAttributes.Keys()
+                    for each attributeKey in attributeKeys
+                        if (mputils.isString(mediaSessionAttributes[attributeKey])) then
+                            eventAttributes[attributeKey] = mediaSessionAttributes[attributeKey]
+                        end if
```

Comment:
Could we have an elseif here for if the value is an integer? Then you could use `ToStr()` like on line 1835 to convert it to a string.

Reply:
@BrandonStalnaker thanks for the catch, just added handling non-string attributes logic